### PR TITLE
Fix documentation, now we are same comments to sync or replicate

### PR DESCRIPTION
**Pull request by kosssi**

### CODE REVIEW

File: docs/_includes/api/replication.html [hunk 1]
```diff
@@ -39,12 +39,12 @@
   retry: true
 }).on('change', function (info) {
   // handle change
-}).on('paused', function (err) {
+}).on('paused', function () {
   // replication paused (e.g. replication up to date, user went offline)
```
Comment (daleharvey) [inline docs/_includes/api/replication.html]: We should add `(err)` to the documentation of both the paused events, it is passed in the event that the replication was paused due to an error

**Comment by daleharvey** [comment]
Having these be consistent would be ideal, good catch. However we shouldnt remove the documentation for the `err` parameter as commented, if you want to fix that then :+1: thanks :)

File: docs/_includes/api/replication.html (hunk 1 above)
Comment (kosssi) [inline docs/_includes/api/replication.html]: Why it's not error event? You know where is the code?

**Comment by daleharvey** [comment]
Thanks so much @kosssi, these changes look great, :+1: and merged in https://github.com/pouchdb/pouchdb/commit/10674e843e9401b13056a58541f9a5107238faf8

**Comment by adamlofts** [comment]
I don't think the Sync object will ever emit an error object in the paused event. Arguably this is a bug in the code rather than the docs.

The relevant lines are:

https://github.com/pouchdb/pouchdb/blob/master/src/sync.js#L67

and

https://github.com/pouchdb/pouchdb/blob/master/src/sync.js#L73

**Comment by kosssi** [comment]
@adamlofts it's not here? https://github.com/pouchdb/pouchdb/blob/99c24a5861e19ac5ae78b35d71c9845a449ba9a0/src/replicate/backoff.js#L36

**Comment by adamlofts** [comment]
@kossi From my memory of looking at this before that fires paused on the replication object with the error but it does not get passed through to the paused event on the Sync object. Could be wrong tho. It was a couple of weeks ago that I looked at it.

**Comment by daleharvey** [comment]
Yeh I think you are right, thats a bug, good catch

**Comment by kosssi** [comment]
I think it's this bug https://github.com/pouchdb/pouchdb/issues/4592 and https://github.com/pouchdb/pouchdb/issues/3714

It's really problematic.

**Comment by kosssi** [comment]
I will test to fix it ;)

**Pull request closed without merging**